### PR TITLE
Integrate data transport from savant-rs

### LINKED ISSUE #229: Integrate data transport from savant-rs
Change data-transport from avro to savant-rs.

### PR DESCRIPTION
**Pull request by tomskikh**

### CODE REVIEW

File: savant/api/builder.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+                (video_object.namespace, video_object.label, track_id)
+            ] = video_object
+
+    for obj_id, obj in enumerate(objects):
+        parent_id = obj.get('parent_object_id')
+        parent_model_name = obj.get('parent_model_name')
+        parent_label = obj.get('parent_label')
+        if parent_id is None or parent_model_name is None or parent_label is None:
+            continue
+        frame.set_parent(
```
Comment (bwsw) [inline savant/api/builder.py]: @tomskikh I added `set_parent_by_id(child_id, parent_id)` which does it more efficiently. Raises an exception if no parent or child exists. Available in `0.1.38`.

File: savant/api/parser.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    value = attribute.values[0]
+    return {
+        'element_name': attribute.namespace,
+        'name': attribute.name,
+        'value': parse_attribute_value(value),
+        'confidence': value.confidence,
+    }
+
+
+def parse_attribute_value(value: AttributeValue):
```
Comment (bwsw) [inline savant/api/parser.py:122]: @tomskikh I made `AttributeValueType` hashable in `0.1.38`.

File: savant/api/builder.py [hunk 3]
```diff
@@ -67,18 +79,16 @@ def add_objects_to_video_frame(
         frame.add_object(video_object, IdCollisionResolutionPolicy.Error)
         track_id = video_object.get_track_id()
```
Comment (bwsw) [inline savant/api/builder.py:80]: @tomskikh , it doesn't look correct. The parent is not related to the track.

The object in metadata must have a reference to the parent. it is not about the track.

File: savant/api/builder.py (hunk 1 above)
Comment (tomskikh) [reply savant/api/builder.py]: Updated savant-rs to 0.1.38

File: savant/api/parser.py (hunk 2 above)
Comment (tomskikh) [reply savant/api/parser.py:122]: Updated savant-rs to 0.1.38

**Review by dorgun — APPROVED** [review]

**Pull request merged**